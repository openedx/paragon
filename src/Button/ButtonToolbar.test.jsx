import React from 'react';
import renderer from 'react-test-renderer';

import { ButtonToolbar } from './index';

describe('<ButtonToolbar />', () => {
  describe('correct rendering', () => {
    it('renders without props', () => {
      const tree = renderer.create((
        <ButtonToolbar />
      )).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
