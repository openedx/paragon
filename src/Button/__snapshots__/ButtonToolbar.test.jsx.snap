// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToolbar /> correct rendering renders without props 1`] = `
<div
  className="btn-toolbar"
  role="toolbar"
/>
`;
